plugin/evm/message/codec.go

```diff
@@ -29,6 +30,8 @@ func init() {
 		// Gossip types
 		c.RegisterType(AtomicTxGossip{}),
 		c.RegisterType(EthTxsGossip{}),
+		c.RegisterType(gossip.PullTxsRequest{}),
+		c.RegisterType(gossip.PullTxsResponse{}),
```

Comment:
We should register these at the end rather than the middle so that they do not impact the typeID used for other message types

Reply:
Ah didn't realize the ordering made a difference here. Will move.